src/assets/stylesheets/components/_input.scss

```diff
@@ -0,0 +1,34 @@
+.form-field {
+  position: relative;
+  padding-bottom: rem(24px);
+
+  &__label {
+    display: flex;
+    flex: none;
+    flex-grow: 0;
+    align-items: center;
+    align-self: stretch;
+    order: 0;
+    width: 20rem;
+    height: 1.25rem;
+    margin-bottom: 0.5rem;
```

Comment:
Change to use rem for these attributes as well 🙏

Reply:
Do you mean change it to use function rem?

Reply:
Fixed in https://github.com/suphanatjarukulgowit/react-nimble-survey/pull/32/commits/895fcad6bdc3a267fd062408d629eadd7bbb2507

Reply:
@suphanatjarukulgowit It seems the changes are disappeared 🙈